tools/file_packager.py

```diff
@@ -621,13 +632,24 @@ def generate_js(data_target, data_files, metadata):
   if options.from_emcc:
     ret = ''
   else:
-    ret = '''
+    if options.modularize:
+      ret = '''
+  export default function loadDataFile(Module) {
+                            '''
```

Comment:
Maybe make this just a single line `ret = 'export default function loadDataFile(Module) {'`